app/controllers/api/databases.php

```diff
@@ -1105,25 +1105,33 @@ function updateAttribute(
     ->param('required', null, new Boolean(), 'Is attribute required?')
     ->param('default', null, new Text(0, 0), 'Default value for attribute when not provided. Cannot be set when attribute is required.', true)
     ->param('array', false, new Boolean(), 'Is attribute an array?', true)
+    ->param('encrypt', false, new Boolean(), 'Encrypt attribute? Encrypting an attribute means that the attribute can not be queried.', true)
     ->inject('response')
     ->inject('dbForProject')
     ->inject('database')
     ->inject('events')
-    ->action(function (string $databaseId, string $collectionId, string $key, ?int $size, ?bool $required, ?string $default, bool $array, Response $response, Database $dbForProject, EventDatabase $database, Event $events) {
+    ->action(function (string $databaseId, string $collectionId, string $key, ?int $size, ?bool $required, ?string $default, bool $array, bool $encrypt, Response $response, Database $dbForProject, EventDatabase $database, Event $events) {
```

Comment:
We also need to add this param to the update String Attribute endpoint. 

Was it discussed to only implement this feature for string attributes?

Reply:
We can not add this for update, otherwise we'd need to migrate all the data when changed. 

It was discussed to allow for strings only - it's the only attribute that it can be used for as it's stored as a string in the database, otherwise validation needed to be skipped. If you really need to encrypt a number, you can still do so as a string